src/components/tx-flow/flows/SignMessage/SignMessage.tsx

```diff
@@ -77,16 +77,27 @@ const createSkeletonMessage = (confirmationsRequired: number): SafeMessage => {
   }
 }
 
-const MessageHashField = ({ label, hashValue }: { label: string; hashValue: string }) => (
-  <>
-    <Typography variant="body2" fontWeight={700} mt={2}>
-      {label}:
-    </Typography>
-    <Typography data-testid="message-hash" variant="body2" component="div">
-      <EthHashInfo address={hashValue} showAvatar={false} shortAddress={false} showCopyButton />
-    </Typography>
-  </>
-)
+const MessageHashField = ({
+  label,
+  hashValue,
+  capitalize,
+}: {
+  label: string
+  hashValue: string
+  capitalize?: boolean
+}) => {
+  const hash = capitalize ? `0x${hashValue.slice(2).toUpperCase()}` : hashValue
```

Comment:
I'll leave it up to you, but we could create a helper to capitalise hex considering this is the third occurance.